lib/logger.js

```diff
@@ -1,10 +1,10 @@
+/* eslint-disable no-magic-numbers */
 /**
  * Module dependencies.
  */
 
-var _ = require('lodash');
-var util = require('./util');
-var ut = require('util');
+const util = require('./util');
+const ut = require('util');
```

Comment:
@milesj I didn't get any trouble from the linter for ordering absolute vs relative imports, do you think we could add a linting rule?